### DISCUSSION TITLE
Hiding default items not working?

### DISCUSSION THREAD

**Discussion by bicaluv** [swiftbar/SwiftBar#300] [Q&A]
Hi!
I started using SwiftBar - which like a lot!!! - and configured in the preferences dialog of SwiftBar that my plugin does not show the About menu item.
After that I realized that I could also use the meta information in my ruby plugins code, using:
`# <swiftbar.hideAbout>true</swiftbar.hideAbout>`
But it looks like that it does not work. Even when changing to `false`, the about menu item is still not shown. Looks like the settings from the plugin specific preferences overwrite the settings coming from the plugin file.

**Comment by melonamin** [comment]
Can you share the plugin you're working on?

**Reply by bicaluv** [reply]
sure:
```
#!/usr/bin/env ruby

# Metadata allows your plugin to show up in the app, and website.
# changed from xbar to SwiftBar: https://github.com/swiftbar/SwiftBar
#
#  <xbar.title>forksync</xbar.title>
#  <xbar.version>v1.1</xbar.version>
#  <xbar.author>bicaluv</xbar.author>
#  <xbar.author.github>bicaluv</xbar.author.github>
#  <xbar.desc>shows if fork needs sync with master</xbar.desc>
#  <xbar.dependencies>ruby</xbar.dependencies>
#  <xbar.abouturl>https://gist.github.com/bicaluv/XXX/</xbar.abouturl>
# <swiftbar.hideAbout>false</swiftbar.hideAbout>
# <swiftbar.hideRunInTerminal>true</swiftbar.hideRunInTerminal>
# <swiftbar.hideLastUpdated>true</swiftbar.hideLastUpdated>
# <swiftbar.hideDisablePlugin>true</swiftbar.hideDisablePlugin>
# <swiftbar.hideSwiftBar>true</swiftbar.hideSwiftBar>

require 'net/http'
require 'json'

GITHUB_SERVER = 'https://api.github.com'
REPO = 'bicaluv/evcc'
ACCESS_TOKEN = 'ghp_xxx' # https://github.com/settings/tokens

begin
    uri = URI("#{GITHUB_SERVER}/repos/#{REPO}/compare/master...evcc-io:master")
    response = ''
    Net::HTTP.start(uri.host, uri.port, :use_ssl => uri.scheme == 'https') do |http|
        request = Net::HTTP::Get.new(uri)
        request['Authorization'] = "token #{ACCESS_TOKEN}"
        response = JSON.parse(http.request(request).body()) # Net::HTTPResponse object
    end

    result = response["ahead_by"]
    if result > 0
        puts "📥"
        puts "---"
        puts "%d commits behind" % [result]
    else
        puts "🟢"
        puts "---"
        puts 'in sync with evcc origin'
    end
rescue
    puts 'GitHub not available| color=red'
    return
end
```

**Reply by melonamin** [reply]
Ok, I read your quiestion again and now get where the confusion is coming from.

When you're tweaking plugin settings in Preferences, all changes are being saved in file extended attributes. These preferences have a priority over what you have in file itself.

If you want SwiftBar to "forget" the extended preferences part, do this `xattr -d com.ameba.SwiftBar <plugin name>`

**Reply by bicaluv** [reply]
Cool, thanks, that was it! Does it make sense to add a "forget extended metadata" button to the UI?
And thanks for the great support and tool!

**Reply by melonamin** [reply]
sure #301
